README.md

```diff
@@ -100,21 +108,22 @@ This feature is used to detect spam in message channels. When a user tries to se
 <h2> Installation and Running </h2>
 
 #### Tools and Libraries Used
-In addition to the packages from [requirements.txt](https://github.com/Ashwinshankar98/TeachersPetBot/blob/main/requirements.txt) which need to be installed, please have the following installed on your machine:
+In addition to the packages from [requirements.txt](https://github.com/tanmaypardeshi/CSC-510-Project2-TeachersPetBot/blob/main/requirements.txt) which need to be installed, please have the following installed on your machine:
+<!-- Are there any changes here? -->
 * [Python 3.9.7](https://www.python.org/downloads/)
 * [Sqlite](https://www.sqlite.org/download.html)
-
+<!-- Are there any changes here? -->
```

Comment:
@ExtremeMachine12 Please let me know what to do here if you know.

Reply:
We can update this to Python 3.10